content/en/docs/Responses/user.md

```diff
@@ -8,12 +8,67 @@ description: >
+  "adminRole": "false",
+  "settingsRole": "true",
+  "downloadRole": "true",
+  "uploadRole": "false",
+  "playlistRole": "true",
+  "coverArtRole": "true",
+  "commentRole": "true",
+  "podcastRole": "true",
+  "streamRole": "true",
+  "jukeboxRole": "true",
+  "shareRole": "false"
+}
 {{< /tab >}}
 {{< tab header="Subsonic" lang="json" >}}
-// TODO
+{
+  "folder": [
+    "1",
+    "3"
+  ],
+  "username": "sindre",
+  "email": "sindre@activeobjects.no",
+  "scrobblingEnabled": "true",
+  "adminRole": "false",
+  "settingsRole": "true",
+  "downloadRole": "true",
+  "uploadRole": "false",
+  "playlistRole": "true",
+  "coverArtRole": "true",
+  "commentRole": "true",
+  "podcastRole": "true",
+  "streamRole": "true",
+  "jukeboxRole": "true",
+  "shareRole": "false"
+}
 {{< /tab >}}
 {{< /tabpane >}}
 
-| Field |  Type | Req. | OpenS. | Details |
-| --- | --- | --- | --- | --- |
+| Field                 | Type           | Req.    | OpenS. | Details                                      |
+| --------------------- | -------------- | ------- | ------ | -------------------------------------------- |
+| `username`            | `string`       | **Yes** |        | Username                                     |
+| `scrobblingEnabled`   | `boolean`      | **Yes** |        | Whether scrobling is enabled for the user    |
+| `maxBitRate`          | `int`          | No      |        |                                              |
+| `adminRole`           | `boolean`      | **Yes** |        | Whether the user is an admin                 |
+| `settingsRole`        | `boolean`      | **Yes** |        | Whether the user is can edit settings        |
+| `downloadRole`        | `boolean`      | **Yes** |        | Whether the user can download                |
+| `uploadRole`          | `boolean`      | **Yes** |        | Whether the user can upload                  |
+| `playlistRole`        | `boolean`      | **Yes** |        | Whether the user can create playlists        |
+| `coverArtRole`        | `boolean`      | **Yes** |        | Whether the user can get cover art           |
+| `commentRole`         | `boolean`      | **Yes** |        | Whether the user can create comments         |
+| `podcastRole`         | `boolean`      | **Yes** |        | Whether the user can create/refresh podcasts |
+| `streamRole`          | `boolean`      | **Yes** |        | Whether the user can stream                  |
+| `jukeboxRole`         | `boolean`      | **Yes** |        | Whether the user can control the jukebox     |
+| `shareRole`           | `boolean`      | **Yes** |        | Whether the user can create a stream         |
+| `videoConversionRole` | `boolean`      | **Yes** |        | Whether the user can convert videos          |
+| `avatarLastChanged`   | `string`       | No      |        | Last time the avatar was changed [ISO 8601]  |
+| `folder`              | Array of `int` | No      |        | Folder ID(s)                                 |
```

Comment:
Should `folder` be using an array of `string` instead?

Reply:
Nope. For some reason, in OG, folder IDs are **integers** instead of strings. That's the only one